config/locales/en.yml

```diff
@@ -37,11 +37,10 @@ en:
           states:
             running: 'Running'
             discovery_report_complete: 'Discovery report completed'
-            preassembly_complete: 'Preassembly completed'
+            preassembly_complete: 'Running'
```

Comment:
Is it worth leaving a comment here about why `preassembly_complete` shares the label `running`?

Reply:
Good idea. Added.